Cargo.toml

```diff
@@ -30,6 +30,7 @@ version = "1.2.2"
 default-features = false
 
 [features]
-default = []
+default = ["kip6"]
```

Comment:
should be default only after the fork?

Reply:
I don't think it's make sense. The binaries will not change until user will use it (it'll optimized out). So the final decision will be still in hands of user and it will depend only on the code user wrote, regardless if `kip6` feature is enabled or disabled.

Reply:
i don't see any point in feature gate if it is on by default

Reply:
I don't see either. That's why I had with question https://github.com/paritytech/parity-ethereum/pull/9357#issuecomment-414639620

Maybe some people would consider it useful, but I can't imagine what kind of case that would be.

Reply:
I don't like the idea to force users to pass this feature in every contract as well.

Reply:
They won't need to pass it until the fork is done with the exception if they use custom chain spec.

And after the fork, it will be the default, so they won't need to pass it again, right?

Reply:
So the idea behind make it the "default" feature is that by "default" user don't have to care either `pwasm-ethereum` has this extern or not (see https://github.com/paritytech/pwasm-ethereum/pull/15#discussion_r211594585)

Reply:
"default" configuration is the kind of configuration that won't allow a user to generate invalid binaries for kovan network

Reply:
> And after the fork, it will be the default, so they won't need to pass it again, right?

I just think that we have to care about this, because again:

> it will depend only on the code user wrote, regardless if kip6 feature is enabled or disabled.

Reply:
> "default" configuration is the kind of configuration that won't allow a user to generate invalid binaries for kovan network

ok, that's a good argument.